webapp/src/ts/app.component.ts

```diff
@@ -646,6 +650,12 @@ export class AppComponent implements OnInit, AfterViewInit {
     });
   }
 
+  private warnOutdatedChrome(): void {
+    if (this.browserDetectorService.isUsingOutdatedChromeBrowser()) {
+      this.modalService.show(BrowserCompatibilityComponent);
```

Comment:
When manually testing, the button text is incorrect for a brief moment. Is this normal?

https://github.com/medic/cht-core/assets/21312057/b06c4b23-6377-4788-9102-c086909f5ff1

Reply:
![image](https://github.com/medic/cht-core/assets/21312057/ab59e3a2-ffc3-410a-a1bc-6ad4819b8050)

Reply:
You can wait for translations like this 
```
private warnOutdatedChrome(): void {
    if (!this.browserDetectorService.isUsingOutdatedChromeBrowser()) {
       return; // Return early to keep code flat
    }
    await this.translationsLoaded;
    this.modalService.show(BrowserCompatibilityComponent);
}
```
The translations can fail to load or be slow.
I guess we still want to show the modal in those scenarios, right?

Reply:
Yes we still want to show the modal. Using your suggestion the translations seems to be loading okay now. Even when I throttle the network. Thanks.